### ISSUE TITLE
Make the version(s) visible again

### ISSUE THREAD

**Issue by stdedos** [YTVanced/VancedManager#535]
**Please check all boxes below by replacing space with** `x`
- [x] I am using the latest version
- [x] This is a suggestion for in the Vanced Manager and ONLY Vanced Manager (NOT Youtube Vanced/Music/microG)

**Suggestion:**

While the new expandable card UI is nice, it is boring to have to extend the cards to see if there is a new version. 

I'd rather extend the description instead (which can be auto-extended, if the app is not installed - if I installed it, I "probably" know what that app is doing)

**Comment by X1nto** [comment]
Thanks for the suggestion, we'll keep that in mind

**Comment by frenchiveruti** [comment]
I agree, new design although nice, is not really that much useful.  
Yes, I admit I open the manager only when I'm notified of an update, but still.

**Comment by X1nto** [comment]
What do you think about this? Basically, the layout is always expanded and descriptions are removed as they're not really needed
![image](https://user-images.githubusercontent.com/48173186/117455447-3ff5fa80-af58-11eb-897e-b1e0d41b3ee1.png)

**Comment by stdedos** [comment]
"Sure", but you have too much wasted vertical space on the _right_. 

Why don't you have a layout like: 
Icon, Name/CurrentVersion, buttons 

and add the "Latest" version on the bottom as a full row (if there is one or "Unavailable"). 

Descriptions can be shown only when it's needed - i.e. when the app is not installed. 😝

**Comment by X1nto** [comment]
> "Sure", but you have too much wasted vertical space on the fight.
> 
> Why don't you have a layout like:
> Icon, Name/CurrentVersion, buttons
> 
> and add the "Latest" version on the bottom as a full row (if there is one or "Unavailable").
> 
> Descriptions can be shown only when it's needed - i.e. when the app is not installed. 😝

This was just a quick mockup I made under 15 minutes just to show off what I had in mind with always expanded layout, it'll be further improved

**Comment by frenchiveruti** [comment]
I have an improvement but I suck at doing graphics.  
What about colors? 
Green = Released matches downloaded version
Yellow = Newer release available 
Red = Not downloaded.  

Because to be honest, as a "normie" I don't care about the version, if I want to see "what's new" I'll probably check the changelogs anyway.  

That way you can save space or come up with some more creative approach.  
Also, about the "names".  
Saying "Youtube Vanced" is kinda... redundant don't you think?  
Why not just leave "Vanced"  
"Vanced Music"  
And "MicroG" for each of them?

**Comment by stdedos** [comment]
> What about colors?
> Green = Released matches downloaded version
> Yellow = Newer release available
> Red = Not downloaded.

Too strong, and _red_ is an error in general in UX ...

> Why not just leave "Vanced"

The application's name is "Vanced Manager", and you want to leave the "Vanced" out of "YouTube Vanced"?
![ ](https://media1.tenor.com/images/824548a44c37fc8984c7c2090ff22f52/tenor.gif?itemid=7852244)

Anyway, IMHO it's not an option (or necessary even).

**Comment by frenchiveruti** [comment]
You seem to be very critical but lacking in the ideas department.

**Comment by stdedos** [comment]
> You seem to be very critical but lacking in the ideas department.  

Both true! And I consider the first one to be a compliment; thank you! 😄 
(PS: I hope my response did not come off as offensive to you, and hence the response)
